docs/get_started.md

```diff
@@ -118,12 +118,28 @@ In CPU mode you can run the demo/webcam_demo.py for example.
 However some functionality is gone in this mode:
 
 - Deformable Convolution
+- Modulated Deformable Convolution
+- ROI pooling
 - Deformable ROI pooling
 - CARAFE: Content-Aware ReAssembly of FEatures
+- SyncBatchNorm
+- CrissCrossAttention: Criss-Cross Attention
+- MaskedConv2d
+- Temporal Interlace Shift
 - nms_cuda
 - sigmoid_focal_loss_cuda
+- bbox_overlaps
 
-So if you try to run inference with a model containing deformable convolution you will get an error.
+So if you try to run inference with a model containing above ops you will get an error. The following table shows the details about not support CPU only inference model
```

Comment:
The following table lists the related methods that cannot inference on CPU due to dependency on these operators.